src/main/java/opendota/Parse.java

```diff
@@ -146,7 +154,20 @@ public UnknownItemFoundException(String message) {
 	private GreevilsGreedVisitor greevilsGreedVisitor;
 	private TrackVisitor trackVisitor;
     private ArrayList<Boolean> isPlayerStartingItemsWritten;
-    
+
+    //Draft stage variable
+    Integer draftStage = 0;
+    int[] draftHeroes = new int[20];
```

Comment:
I don't think these arrays actually need to be globally scoped here. . . can't you just create them in the section below where you use them?

Reply:
Im not sure I understand. The bans and picks come through in new OnTickStarts so I thought I had to declare them globally so that they would be updated.

Reply:
You can create a new array in each call to onTickStart